packages/perseus/src/widgets/group/group.tsx

```diff
@@ -28,12 +28,15 @@ import type {
 import type {GroupPromptJSON} from "../../widget-ai-utils/group/group-ai-utils";
 import type {
     PerseusGroupWidgetOptions,
+    ShowSolutions,
     UserInputArray,
     UserInputMap,
 } from "@khanacademy/perseus-core";
 
 type RenderProps = PerseusGroupWidgetOptions; // exports has no 'transform'
-type Props = WidgetProps<RenderProps>;
+type Props = WidgetProps<RenderProps> & {
+    showSolutions?: ShowSolutions; // Provided by the renderer
```

Comment:
Instead of selectively adding `showSolutions` to the widgets that use it, I think it'd be better to add it to `UniversalWidgetProps`. That's what is happening in reality (the Renderer [passes](https://github.com/Khan/perseus/blob/4ac22bdadcdad844ed9964485ef727de53c6dfa7/packages/perseus/src/renderer.tsx#L596) it to each and every widget it renders).